k9mail/src/main/java/com/fsck/k9/search/SqlQueryBuilder.java

```diff
@@ -72,6 +75,15 @@ private static void buildWhereClauseInternal(Account account, ConditionsTreeNode
                     }
                     break;
                 }
+                case MESSAGE_CONTENTS: {
+                    String fulltextQueryString = condition.value;
+                    if (condition.attribute != Attribute.CONTAINS) {
+                        Log.e(K9.LOG_TAG, "message contents can only be matched!");
```

Comment:
I wonder if it's worth throwing an error here as otherwise the user will think it's broken rather than unsupported?